lib/saml_idp/assertion_builder.rb

```diff
@@ -88,7 +88,7 @@ def encrypt(opts = {})
     def asserted_attributes
       if principal.respond_to?(:asserted_attributes)
         principal.send(:asserted_attributes)
-      elsif !config.attributes.nil? && !config.attributes.empty?
```

Comment:
I'm inclined to head the code towards not relying on ActiveSupport for a number of reasons. Please remove this and I'll merge the updated documentation. Thanks for the add, sorry for the late response.